### DISCUSSION TITLE
What licenses to use for UHC public repos

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/analytics-corner#17] [Q&A]
general question to  

@Drexel-UHC/uhc_research_data_core 
@usamabilal  @aquistbe @alinasmahl1  (feel free to tag others you have some insight)

which licenses do people use for public access repositories? 

## Some popular ones to choose from

   - **WTFPL** license (google it)
   - **MIT** License is a permissive license that allows users to modify, distribute, and use the software for any purpose, including for commercial use. 
   - **GNU GPL**: very permissive - commercial + non-commercial but requires all modifications to be under GNU GPL (also be very permissive)
   -  **CCBY**:  very permissive - commercial + non-commercial but requiers attribution for credit. 
   - **AFL** very permissive - commercial + non-commercial but requiers attribution for credit.

**Comment by usamabilal** [comment]
I think we had this discussion before (somewhere in here or over email; with Andrea i think). A tricky thing is that those are all code licenses, not data, right?

**Reply by ran-codes** [reply]
Yea... we ended up with CCBY for SLAURBAL data. Kari and Matt wanted some recommendations for the UHC drexel github organization code. so we need some sort of guidelines for people making repos to guide which license they should use.

**Reply by aquistbe** [reply]
I think CCBY is probably one of the most recognizable, i believe one or more of the vital registry data in salurbal were under ccby. There are many variations of CC licenses https://creativecommons.org/choose/. Question is if you want to add the non-commercial or not.
